src/vs/workbench/browser/parts/titlebar/media/menubarControl.css

```diff
@@ -0,0 +1,61 @@
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.monaco-workbench .menubar > .menubar-menu-button,
+.monaco-workbench .menubar .toolbar-toggle-more {
+	color: var(--vscode-titleBar-activeForeground);
+}
+
+.monaco-workbench .activitybar .menubar.compact > .menubar-menu-button,
+.monaco-workbench .activitybar .menubar.compact .toolbar-toggle-more {
+	color: var(--vscode-activityBar-inactiveForeground);
+}
+
+.monaco-workbench .activitybar .menubar.compact > .menubar-menu-button.open,
+.monaco-workbench .activitybar .menubar.compact > .menubar-menu-button:focus,
+.monaco-workbench .activitybar .menubar.compact:not(:focus-within) > .menubar-menu-button:hover,
+.monaco-workbench .activitybar .menubar.compact  > .menubar-menu-button.open .toolbar-toggle-more,
+.monaco-workbench .activitybar .menubar.compact > .menubar-menu-button:focus .toolbar-toggle-more,
+.monaco-workbench .activitybar .menubar.compact:not(:focus-within) > .menubar-menu-button:hover .toolbar-toggle-more {
+	color: var(--vscode-activityBar-foreground);
+}
+
+.monaco-workbench .menubar.inactive:not(.compact) > .menubar-menu-button,
+.monaco-workbench .menubar.inactive:not(.compact) > .menubar-menu-button .toolbar-toggle-more  {
+	color: var(--vscode-titleBar-inactiveForeground);
+}
+
+.monaco-workbench .menubar:not(.compact) > .menubar-menu-button.open,
+.monaco-workbench .menubar:not(.compact) > .menubar-menu-button:focus,
+.monaco-workbench .menubar:not(:focus-within):not(.compact) > .menubar-menu-button:hover,
+.monaco-workbench .menubar:not(.compact) > .menubar-menu-button.open .toolbar-toggle-more,
+.monaco-workbench .menubar:not(.compact) > .menubar-menu-button:focus .toolbar-toggle-more,
+.monaco-workbench .menubar:not(:focus-within):not(.compact) > .menubar-menu-button:hover .toolbar-toggle-more {
+	color: var(--vscode-menubar-selectionForeground);
+}
+
+.monaco-workbench .menubar:not(.compact) > .menubar-menu-button.open .menubar-menu-title,
+.monaco-workbench .menubar:not(.compact) > .menubar-menu-button:focus .menubar-menu-title,
+.monaco-workbench .menubar:not(:focus-within):not(.compact) > .menubar-menu-button:hover .menubar-menu-title {
+	background-color: var(--vscode-menubar-selectionBackground);
+}
+
+.monaco-workbench .menubar > .menubar-menu-button:hover .menubar-menu-title  {
+	outline: dashed 1px;
+}
+
+.monaco-workbench .menubar > .menubar-menu-button.open .menubar-menu-title,
+.monaco-workbench .menubar > .menubar-menu-button:focus .menubar-menu-title {
+	outline: solid 1px;
+}
+
+.monaco-workbench .menubar > .menubar-menu-button.open .menubar-menu-title,
+.monaco-workbench .menubar > .menubar-menu-button:focus .menubar-menu-title,
+.monaco-workbench .menubar > .menubar-menu-button:hover .menubar-menu-title {
+	outline-color: var(--vscode-menubar-selectionBorder);
```

Comment:
the selection border is not always set and there for the collection of rules around outline should not be set if it is false see https://github.com/microsoft/vscode/issues/165169#issuecomment-1345213440

Reply:
Indeed, I overlooked that, I have temporarily added back the collection of this variable, and if there is a good way to modify it in the future, it can be modified together